app/main.py

```
@@ -0,0 +1,55 @@
+# Delete a ToDo item
+@app.delete("/todos/{todo_id}", status_code=204)
+async def delete_todo(todo_id: int, db: AsyncSession = Depends(get_db)) -> None:
+    await crud.delete_todo(db, todo_id)
+
+
+# Update a ToDo item
+@app.put("/todos/{todo_id}")
+async def update_todo(
+    todo_id: int,
+    todo: schemas.ToDoUpdate,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)